copt/constraint.py

```diff
@@ -20,52 +20,43 @@ def __call__(self, x):
         else:
             return np.infty
 
-    def prox(self, x, step_size):
+    def prox(self, x, step_size=None):
         return euclidean_proj_l1ball(x, self.alpha)
 
-    def lmo(self, u, x):
-        """Return s - x, s solving the linear problem
+    def lmo(self, u, x, active_set=None):
```

Comment:
please describe all returned parameters, if possible in the format

```
Parameters
---------------
u : bla

v : bla
```

(you can copy-paste)

Reply:
and I know it was not properly documented before, but let's use this as a chance to do things right ;-)